server/src/main/scala/org/http4s/server/middleware/Logger.scala

```diff
@@ -23,11 +24,11 @@ object Logger {
       )
     )
 
-  def logMessage[F[_], A <: Message[F]](message: A)(
+  def logMessage[M[_[_]], F[_]](message: M[F])(
```

Comment:
nothing wrong with this, just wanted to point out that getting rid of the covariant bound here looks great :+1: